curl_cffi/aio.py

```diff
@@ -53,19 +121,22 @@ def timer_function(curlm, timeout_ms: int, clientp: Any):
 def socket_function(curl, sockfd: int, what: int, clientp: Any, data: Any):
     async_curl = ffi.from_handle(clientp)
     loop = async_curl.loop
+    selector = _get_selector(loop)
```

Comment:
If _get_selector has been called in AsyncCurl.__init__, so we have to call it again here? since this callback is used quite often.